crates/astria-sequencer/src/app/mod.rs

```diff
@@ -793,15 +903,33 @@ impl App {
         }
 
         ensure!(
-            finalize_block.txs.len() >= 2,
-            "block must contain at least two transactions: the rollup transactions commitment and
-             rollup IDs commitment"
+            finalize_block.txs.len() >= 3,
+            "block must contain at least three transactions: the extended commit info, the rollup \
+             transactions commitment and rollup IDs commitment"
         );
 
+        let extended_commit_info_bytes = &finalize_block.txs[0];
```

Comment:
This is correct but prone to fail when refactored; this makes it a bit clearer for the reader what to look for:
```suggestion
        let extended_commit_info_bytes = finalize_block.get(0).expect("asserted length above");
```